app/Helpers/IconHelper.php

```diff
@@ -184,7 +184,9 @@ public static function icon($type) {
                 return 'fa-regular fa-id-card';
             case 'department' :
                 return 'fa-solid fa-building-user';
-
+            case 'note':
```

Comment:
Not sure what this is doing?

Reply:
this is called at line 260 on the hardware.view blade
`<x-icon type="note" />`
it gives us the icon on the Add Note button
<img width="459" alt="Screenshot 2024-12-09 at 9 25 13 PM" src="https://github.com/user-attachments/assets/d96ab9a3-9d8e-4266-9c04-d7592fd2dcc8">

Reply:
this mimics the rest of our icon typing such as
```
case 'assets':
case 'asset':
     return 'fas fa-barcode';
case 'accessories':
case 'accessory':
     return 'far fa-keyboard';
```